### PR TITLE
Add nrf53 targets

### PR DESCRIPTION
**Pull request by KKopyscinski**
Added btshell to compile for app core and blehci to compile for net core

### CODE REVIEW

**Comment by KKopyscinski** [comment]
with this configurations it requires https://github.com/apache/mynewt-core/pull/2488 to build btshell

**Comment by KKopyscinski** [comment]
@sjanc could you look at this?

File: mynewt-core-targets/nordic_pca10095_blehci/syscfg.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+syscfg.vals:
+    BLE_MAX_CONNECTIONS: 4
+
+    #BLE_HCI_UART_BAUD: 1000000
+    #BLE_HCI_UART_FLOW_CTRL: HAL_UART_FLOW_CTL_RTS_CTS
+    BLE_HCI_UART_BAUD: 115200
+    BLE_HCI_UART_FLOW_CTRL: HAL_UART_FLOW_CTL_NONE
```
Comment (sjanc) [inline mynewt-core-targets/nordic_pca10095_blehci/syscfg.yml:7]: BLE_HCI_UART_* are not needed, as those are for UART transport only

File: mynewt-core-targets/nordic_pca10095_btshell/syscfg.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+syscfg.vals:
+    BSP_NRF5340_NET_ENABLE: 1
```
Comment (sjanc) [inline mynewt-core-targets/nordic_pca10095_btshell/syscfg.yml:2]: new line

File: mynewt-core-targets/nordic_pca10095_blehci/syscfg.yml (hunk 1 above)
Comment (sjanc) [reply mynewt-core-targets/nordic_pca10095_blehci/syscfg.yml:7]: also enable all features, below should be enough:
    BLE_LL_CFG_FEAT_DATA_LEN_EXT: 1
    BLE_LL_CFG_FEAT_LE_2M_PHY: 1
    BLE_LL_CFG_FEAT_LE_CODED_PHY: 1
    BLE_LL_CFG_FEAT_LL_PRIVACY: 1
    BLE_LL_CONN_INIT_MAX_TX_BYTES: 251
    BLE_LL_CONN_INIT_SLOTS: 4
    BLE_LL_DTM: 1
    BLE_LL_DTM_EXTENSIONS: 1
    BLE_LL_VND_EVENT_ON_ASSERT: 1
    BLE_MAX_CONNECTIONS: 5
    BLE_MAX_PERIODIC_SYNCS: 5
    BLE_MULTI_ADV_INSTANCES: 5
    BLE_PERIODIC_ADV: 1
    BLE_PERIODIC_ADV_SYNC_TRANSFER: 1
    BLE_VERSION: 51

**Comment by KKopyscinski** [comment]
@sjanc done

**Review by sjanc — APPROVED** [review]

**Pull request merged**